### ISSUE TITLE
Extra information 

### ISSUE THREAD

**Issue by watashinosoulsociety** [Malopieds/InnerTune#407]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

So basically I want an option where I could like when I click details I get more credits of the song I'm listening to like the producer and the date/day the song got released from 

### Why do you want this feature?

It'd be real easy to know bout the other artists and get into their discography if possible

**Comment by WhoSaysViolet** [comment]
Yeah, I want this just like what simpmusic has done, but I dont use simptube because it's slow and the lyrics are so slow too
